.github/workflows/jooq.yml

```
@@ -0,0 +1,79 @@
+
+      - name: 생성 결과 설정
+        if: always()
+        id: result
+        run: |
+          if [ "${{ steps.jooq-cache.outputs.cache-hit }}" = "true" ]; then
+            echo "status=cached" >> $GITHUB_OUTPUT
+          elif [ "${{ steps.jooq-generation.outcome }}" = "success" ]; then
+            echo "status=generated" >> $GITHUB_OUTPUT
+          else
+            echo "status=failed" >> $GITHUB_OUTPUT
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)